svc/pkg/ds/src/workflows/server/destroy.rs

```diff
@@ -84,7 +83,7 @@ struct UpdateDbOutput {
 #[activity(UpdateDb)]
 async fn update_db(ctx: &ActivityCtx, input: &UpdateDbInput) -> GlobalResult<UpdateDbOutput> {
 	// Run in transaction for internal retryability
-	rivet_pools::utils::crdb::tx(&ctx.crdb().await?, |tx| {
+	let db_output = rivet_pools::utils::crdb::tx(&ctx.crdb().await?, |tx| {
 		let ctx = ctx.clone();
 		let server_id = input.server_id;
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +91]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->can you double check that this is using an index with the stored columns configured correctly?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/1081?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->